android/src/main/java/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -195,6 +205,13 @@ class PagerViewViewManager : ViewGroupManager<ViewPager2>() {
     }
   }
 
+  private fun refreshViewChildrenLayout(view: View) {
+    view.measure(
```

Comment:
Please use below code (wrap it into view.post)
```android
    view.post {
      view.measure(
              View.MeasureSpec.makeMeasureSpec(view.width, View.MeasureSpec.EXACTLY),
              View.MeasureSpec.makeMeasureSpec(view.height, View.MeasureSpec.EXACTLY))
      view.layout(view.left, view.top, view.right, view.bottom)
    }
```

Reply:
Done!